backend/src/ee/services/saml-config/saml-config-service.ts

```diff
@@ -338,7 +338,8 @@ export const samlConfigServiceFactory = ({
             email,
             firstName,
             lastName,
-            authMethods: [AuthMethod.EMAIL]
+            authMethods: [AuthMethod.EMAIL],
+            isGhost: false
```

Comment:
Not setting the field will lead to type errors. I suspect this is because I set the field to `notNullable`. Could this be an issue with the type system?

cc @akhilmhdh